### ISSUE TITLE
I can't send emails

### ISSUE THREAD

**Issue by fesnavarro** [LukeSmithxyz/mutt-wizard#591]
I'm having trouble while sending e-mails.

Those are the messages I'm receiving when trying to send my e-mails through accounts I have configured.

yahoo
msmtp: envelope from address whatever@yahoo.com not accepted by the server
msmtp: server message: 554 Email rejected
msmtp: could not send mail (account whatever@yahoo.com from /home/whatever/.msmtprc)

or gmail

msmtp: envelope from address whatever@gmail.com not accepted by the server
msmtp: server message: 530 5.7.0 Must issue a STARTTLS command first. z7sn5787689pfg.1 - gsmtp
msmtp: could not send mail (account whatever@gmail.com from /home/whatever/.msmtprc)

or hotmail

msmtp: envelope from address whatever@hotmail.com not accepted by the server
msmtp: server message: 451 5.7.3 STARTTLS is required to send mail [XCVBR02CA0143.ausprd01.prod.outlook.com]
msmtp: could not send mail (account whatever@hotmail.com from /home/whatever/.msmtprc)

**Comment by fesnavarro** [comment]
Because I'm using Ubuntu my config file was in a different path. After link it to generated by wizard I had to write the default configuration again. Now it's working.

**Comment by LukeSmithxyz** [comment]
Where specifically was the link required? There is a line in the mutt-wizard that is _supposed_ to link the generated config to `~/.msmtprc`, but if that is the place you put it, obviously it didn't work as expected.

**Comment by fesnavarro** [comment]
I had to manual link "ln -sv /home/$USER/.config/msmtp/config /home/$USER/.msmtprc". I also had to write the default configuration again, probably the script filed a empty file on my .config folder.
I don't know if matters but the funny thing I can remember about it was
using cat on ~/.msmtprc was giving me an output (the default one) but
anytime I was using an editor (vim, nvim or even nano) it was showing
the linked content from .config. Weird right? lol Never saw it before,
but ok, probably some weird cache.
